dom.bs

```diff
@@ -3594,6 +3625,67 @@ run these steps:
 <ol>
  <li><p>Assert: either <var>addedNodes</var> or <var>removedNodes</var> <a for=set>is not empty</a>.
 
+ <li>
+  <p>If either <var>options</var>'s' {{MutationObserverInit/elementFilter}} or
+  {{MutationObserverInit/elementByAttributeFilter}} is present, then:
+
+  <ol>
+   <li><p>Let <var>filteredAddedNodes</var> be an empty <a for=/>list</a>.
+
+   <li><p>Let <var>filterRemovedNodes</var> be an empty <a for=/>list</a>.
+
+   <li>
+    <p>For each <var>node</var> in <var>addedNodes</var></a>:
+
+    <ol>
+     <li><p>Let <var>nodeAttributesList</var> be the set of local names of <var>node</var>'s
+     attributes.
```

Comment:
"local names" and "attributes" should xref the relevant definitions. (We should also have a test with two attributes that have the same local name and see if this is all working as designed.)